guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc

```diff
@@ -22,7 +22,6 @@ You can also register Atomic Hosts to {ProjectServer} or {SmartProxyServer}.
 
 Use one of the following procedures to register a host:
 
-* xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]
```

Comment:
Looks like there are multiple occurrences, did you mean to remove them all?

```
[lzap@x1 guides]$ ag "registering-a-host-to-project-using-the-global-registration-template_managing-hosts"
doc-Content_Management_Guide/topics/QuickStart.adoc
154:Use your activation key to register a host to the Content View. For more information on registering a host, see {ManagingHostsDocUR}registering-a-host-to-project-using-the-global-registration-template_managing-hosts

doc-Managing_Hosts/topics/using_insights_with_satellite_hosts.adoc
12:For more information, see {ManagingHostsDocURL}registering-a-host-to-project-using-the-global-registration-template_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in the _Managing Hosts_ guide.
```

I do not understad this change, we have the same in master too. Are all these links are supposed to link here:

```
[id="registering-a-host_managing-hosts"]
= Registering a Host to {ProjectName}
```

Reply:
Asked here: https://github.com/theforeman/foreman-documentation/pull/359/files#r714105415